SQL/2016-03-11-GenomicBrowser-CPG.sql

```diff
@@ -0,0 +1,66 @@
+-- ALTER TABLE genome_loc ADD Description text, MODIFY Strand enum ('forward', 'reverse');
+ALTER TABLE genotyping_platform ADD UNIQUE (Name);
+
+DROP TABLE IF EXISTS `genomic_sample_candidate_rel`;
+CREATE TABLE `genomic_sample_candidate_rel` (
+  `sample_label` varchar(100) NOT NULL,
+  `candidate_id` int unsigned NOT NULL,
```

Comment:
CandID int(6).  
sample_label - ok here while we're awaiting biobanking's table to flesh out sample data (sampleID, sample_label).  On the understanding that sample_label will be unique and essentially be composed of identifiers (e.g. PSCID_CandID...)